config/user_config.go

```diff
@@ -84,9 +84,18 @@ func (cfg *UserConfig) resolve(path string) string {
 	if strings.HasPrefix(path, "~"+string(os.PathSeparator)) {
 		return strings.Replace(path, "~", cfg.Home, 1)
 	}
+	// if using "~/" on Windows
+	if strings.HasPrefix(path, "~/") {
+		path = strings.Replace(path, "~/", "", 1)
+		return filepath.Join(cfg.Home, path)
+	}
```

Comment:
I think if we have this, then we can delete lines 84-86, as your version should work on all platforms, right?